### PR TITLE
[ch28811] update libs: fix collation error on inspectschema cmd

### PR DESCRIPTION
**Pull request by emaciupe**
[ch28811]

### CODE REVIEW

File: src/etools_datamart/apps/multitenant/management/commands/inspectschema.py [hunk 1]
```diff
@@ -179,7 +178,7 @@ def strip_prefix(s):
                     if att_name == 'id' and extra_params == {'primary_key': True}:
                         if field_type == 'AutoField(':
                             continue
-                        elif field_type == 'IntegerField(' and not connection.features.can_introspect_autofield:
+                        elif field_type == 'IntegerField(' and not connection.features.introspected_field_types:
```
Comment (domdinicola) [inline src/etools_datamart/apps/multitenant/management/commands/inspectschema.py:181]: ok

File: src/etools_datamart/apps/multitenant/postgresql/introspection.py [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 from .utils import raw_sql
 
-fields = FieldInfo._fields
-if 'default' not in fields:
-    fields += ('default',)
-
-FieldInfo = namedtuple('FieldInfo', fields)
+FieldInfo = namedtuple(
+    'FieldInfo',
+    'name type_code display_size internal_size precision scale null_ok '
```
Comment (domdinicola) [inline src/etools_datamart/apps/multitenant/postgresql/introspection.py:10]: do we need to list all the fields here?

Comment (domdinicola) [reply src/etools_datamart/apps/multitenant/postgresql/introspection.py:10]: there's no way to retrieve them?

Comment (domdinicola) [reply src/etools_datamart/apps/multitenant/postgresql/introspection.py:10]: https://github.com/django-tenants/django-tenants/pull/693/files

File: src/etools_datamart/apps/multitenant/management/commands/inspectschema.py (hunk 1 above)
Comment (emaciupe) [reply src/etools_datamart/apps/multitenant/management/commands/inspectschema.py:181]: replaced starting with django 3.2

**Pull request merged**